src/pages/ProgramAdminPage/ProgramCoverForm.tsx

```diff
@@ -175,14 +203,16 @@ const ProgramCoverForm: React.VFC<{
         >
           <div className="d-flex align-items-center">
             <ImageUploader
-              file={coverImage}
+              file={coverMedia}
               initialCoverUrl={coverDefaultUrl}
               onChange={file => {
-                setCoverImage(file)
+                setCoverMedia(file)
                 setIsUseOriginSizeCoverImage(false)
               }}
+              customName={formatMessage(commonMessages.ui.uploadMedia)}
+              customImg={!isShowOriginSizeCover ? EmptyVideoCover : undefined}
             />
-            {(!isEmpty(coverDefaultUrl) || coverImage) && (
+            {isShowOriginSizeCover && (
```

Comment:
If it is confirmed that this `checkbox` does not need to be displayed in the video format, you can using `coverMidea.type` without use setState.